crates/codegen/parser_templates/src/typescript/parser_output.rs

```diff
@@ -15,7 +15,7 @@ pub struct ParserOutput {
 
 #[napi]
 impl ParserOutput {
-    #[napi(ts_return_type = "RuleNode | null")]
+    #[napi(ts_return_type = "RuleNode | TokenNode | null")]
```

Comment:
nit: I suggest adding the prefix `chore: ` to the commit message/PR title for now.

Soon, we will have a linter for this kind of issues.

Reply:
done